Countries/ListViewController.swift

```diff
@@ -11,50 +11,48 @@ enum TableConstants {
     static let heightForRow: CGFloat = 100.0
 }
 
-class ListViewController: UIViewController {
+class ListViewController: UITableViewController {
```

Comment:
:)

FYI: It was possible to fix your issue with table view without migrating to UITableViewController

Reply:
Yes, but there is no reason to not using TableViewController in this project) so, I've decided to migrate to it